changelog.md

```diff
@@ -2,21 +2,33 @@
 
 ## Changes in upcoming release (`dev` branch) 
 
+
+### New features
+
+- Added support for the report system to:
+    - `maxibin2`
```

Comment:
maxibin2? or maxbin2?

Reply:
Captain Diogo always checks the changelog... My bad X) Fixed